src/generator-enums.cs

```diff
@@ -122,11 +122,11 @@ void GenerateEnum (Type type)
 			foreach (var kvp in fields) {
 				var f = kvp.Key;
 				var fa = kvp.Value;
-				// the attributes (availability and field) are important for our tests
-				PrintPlatformAttributes (f);
```

Comment:
Why not keep this info on the field? Aren't fields also checked for null on our tests?